tests/ui/mismatched_types/closure-ref-114180.rs

```diff
@@ -0,0 +1,8 @@
+// check-fail
```

Comment:
Wait, actually, can you make this into `// run-rustfix` test? (also `// check-fail` is the default iirc, you don't need it)

Reply:
Unfortunately no, since the current suggestion is not enough to fix the issue; but applying it will lead to a another suggestion which fixes the issue.

Reply:
oof, that's unfortunate